src/nvapi_d3d.cpp

```diff
@@ -6,14 +6,34 @@ extern "C" {
 
     NvAPI_Status __cdecl NvAPI_D3D_GetObjectHandleForResource(IUnknown *pDevice, IUnknown *pResource, NVDX_ObjectHandle *pHandle) {
         static bool alreadyLogged = false;
-        return NoImplementation("NvAPI_D3D_GetObjectHandleForResource", alreadyLogged);
+        constexpr auto n = __FUNCTION__;
```

Comment:
Oh nice, thanks, I didn't know about `__FUNCTION__`.  Could you please keep these new methods consistent and use the method name typed out? Additionally please open an issue (or a PR:)) for replacing those typed out function names everywhere with `__FUNCTION__`.